src/storage/src/hummock/sstable/block.rs

```diff
@@ -450,7 +450,13 @@ impl BlockBuilder {
         if self.entry_count > 0 {
             debug_assert!(!key.is_empty());
             debug_assert_eq!(
-                KeyComparator::compare_encoded_full_key(&self.last_key[..], &key[..]),
+                KeyComparator::compare_full_key_without_table_id(
```

Comment:
The original code seems more consice so I suggest we keep the original one and we don't need to introduce the `compare_full_key_without_table_id` function only for debug_seert.